scripts/aggregate-json-files.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+instances=(brb dbc nbc)
+environments=(dev ref)
+
+workflow_name=$1
+brb_instance=$2
+dbc_instance=$3
+nbc_instance=$4
+workspace_path=$5
+
+check_environment() {
+  local url="$1"
+
+  if [[ $url == *"staging"* || $url == *"schulportal"* ]]; then
+    echo "ref"
+  else
+    echo "dev"
+  fi
+}
+
+if [[ $workflow_name == *"manual"* ]]; then
+  echo "This is a manual workflow"
+  echo "TAG=tag:stable:ci" >>$GITHUB_OUTPUT
+  echo "$TAG"
+elif [[ $workflow_name == *"automatic"* || $workflow_name == *"scheduled"* ]]; then
+  echo "This is an automatic or scheduled workflow"
+  echo "TAG=tag:stable:ci" >>$GITHUB_OUTPUT
+  echo "$TAG"
+else
+  echo "This is a remote workflow"
+  echo "TAG=tag:stable:pr:ci" >>$GITHUB_OUTPUT
+  echo "$TAG"
+fi
+
+brb_env=$(check_environment "$brb_instance")
+dbc_env=$(check_environment "$dbc_instance")
+nbc_env=$(check_environment "$nbc_instance")
+
+if [[ $brb_env == "ref" || $dbc_env == "ref" || $nbc_env == "ref" ]]; then
+  environment="ref"
+else
+  environment="dev"
+fi
```

Comment:
Move this with the requeied Imput URL to an own step in this job who set the enviorment as GITHUB_ENV this woud make in the main workflow things easyser reduce the 1PW Calls to only the 3 who are need

Reply:
fixed